simpleLooping/twoNumberGreatestCommonDivisor.py

```diff
@@ -6,3 +6,23 @@
 #   twoNumberGreatestCommonDivisor(24, 36) # return 12
 #   twoNumberGreatestCommonDivisor(-6, 12) # return 6
 #   twoNumberGreatestCommonDivisor(-6, -3) # return 3
+def twoNumberGreatestCommonDivisor(num1: int, num2: int) -> int:
+    if num1 == 0 and num2 == 0:
+        return -1
+    
+    if num1 < 0:
+        num1 = -num1
+    
+    if num2 < 0:
+        num2 = -num2
+    
+    if num1 < num2:
+       num1, num2 = num2, num1
+    
+    r = num1 % num2
+
```

Comment:
Does

    reminder = num1 % num2

not part of the while loop? this is kind of initialization for iterable if you think about it.